util/test/chplExperimentGatherUtils/boilerplate.bash

```diff
@@ -0,0 +1,23 @@
+# Source this script at the top of your gather scripts to automatically:
```

Comment:
maybe name this file `common.bash`

Reply:
How about `prelude`? (I guess `common` sounds like a bunch of common utility functions, and `boilerplate` sounds like the stuff you have to repeat that you can't just factor out into another file). Could also do `preamble`.

Reply:
Either is OK by me (it took me a second to bring this to my cache :D)

Reply:
A response every couple of months should be close enough to keep temporal locality right?

(I'm hoping to get this stuff in once we're past the release and things have reopened; plus I recently used it to remeasure ChOp stuff).